app/components/user-cards.js

```diff
@@ -1,40 +1,87 @@
 import Component from '@ember/component';
-import { computed } from '@ember/object';
+import EmberObject, { computed } from '@ember/object';
+
 
 export default Component.extend({
 
     selected_option:null,
+    sort_option:null,
+    sort_order:null,
+    isSelected:null,
+    isSelectedOrder:null,
+    sortfieldList:null,
+    sortorderlist:null,
     
-    teamslist : computed ('selected_option',function(){
-        if(this.selected_option!='All Employees')
-            return this.get('members').filterBy('team',this.get('selected_option'))
-        else
-            return this.get('members').filterBy('team')
-        
+    teamslist : computed ('selected_option','sort_option','sort_order',function(){
+    
+        if(this.sort_order==='Ascending'){
+            if(this.selected_option!='All Employees')
+            
+                return this.get('members').filterBy('team',this.get('selected_option')).sortBy(this.get('sort_option'))
+            else
+                return this.get('members').filterBy('team').sortBy(this.get('sort_option'))
+        }
+        else {
+            if(this.selected_option!='All Employees')
+                return this.get('members').filterBy('team',this.get('selected_option')).sortBy(this.get('sort_option')).reverse()
+            else
+                return this.get('members').filterBy('team').sortBy(this.get('sort_option')).reverse()
+        }
     }) ,
 
     teams: computed ('members.[]',function(){
-        
-         let teamNames =  this.members.mapBy('team').uniq();
-         teamNames.push('All Employees');
-         return teamNames;
-        
+        let teamNames =  this.members.mapBy('team').uniq();
+        teamNames.push('All Employees');
+        return teamNames;
+       
+    }),
+
+    sortfieldArrayList: computed('sortfieldList.[]',function(){
+        return this.sortfieldList.filterBy('title')
+    }),
+
+    sortorderArrayList:computed('sortorderlist.[]',function(){
+        return this.sortorderlist.filterBy('title')
     }),
```

Comment:
What are we doing here ? Is it really required ?

Reply:
This code piece has been removed.